balance.md

```diff
@@ -0,0 +1,36 @@
+# Balance of various aspects of the game
+
+**summary** :
+
+The driving goal of balance in DDA is to support versimilitude in the scenario.  In other words we want the right *things* to be present to match reality, but also when those things are used to accomplish goals, they should be appoximately as effective at doing so as their real-world counterparts.
+
+Some examples:  
+Modern vehicles are quite easy to operate and effective at their core competencies of moving people and items around on reasonably well-prepared surfaces, as such they are incredibly valuable and powerful tools in the game as well.
+Guns deal FAR more damage than elastic projectile launchers like bows and crossbows.
+Melee weapons consume considerable effort to operate and are generally significantly more dangerous to use than ranged weapons.
+Some foods are healthy and plentiful, when those foods are in season survival becomes easy.
+
+This concept is backstopped by the concept that reality has a reasonable and interesting set of balance tradeoffs, and approaching this set of tradeoffs makes for a fun game.
+
+## Melee
+The player can use *highly* suboptimal unarmed strikes with no equipment, which deals low damage per second and comes with a high stamina cost.  
+Improvised melee weapons such as sticks or primitive spears greatly improve on unarmed, to the point where unarmed can be considered a dead end.  
+Improvised weapons suffer from relative fragility, and should be replaced with a purpose built melee weapon as soon as possible.  
+Repurposed tools such as hammers and machetes, as well as survivor-crafted equivalents represent an intermediate tier of melee weapon that can be relied on long-term, but are still outclassed by purpose-built melee weapons.  
+At the pinnnacle of melee weapons are purpose-built melee weapons such as swords, robust spears, polearms and axes.  These weapons will reliably outperform all kinds of repurposed weapons, but will have some differentiation in how effective they are based on the situation.  
+Martial arts training and skill can extend the usefullness of melee combat of all kinds, but not to the point where melee combat can become the player's sole form of combat.  
+The intended niche for melee combat of all kinds is to conserve other resources when dealing with relatively low-threat opponents, or as a defense when caught unprepared.  
```

Comment:
Enemies that are difficult (or have side effects) to use guns against - possibility?

Reply:
Also close quarters fighting when you need to be silent, for example in a lab.